crypto/src/secp256k1.rs

```diff
@@ -327,7 +335,13 @@ impl TryInto<Secp256k1PublicKey> for Secp256k1PublicKeyBytes {
 
 impl From<Secp256k1PublicKey> for Secp256k1PublicKeyBytes {
     fn from(pk: Secp256k1PublicKey) -> Secp256k1PublicKeyBytes {
-        Secp256k1PublicKeyBytes::new(pk.pubkey.to_bytes())
+        Secp256k1PublicKeyBytes::new(
```

Comment:
sorry was referring to this line re oncecell, to avoid duplicated code with my previous comment

Reply:
Yep, on this one, we should use `as_ref()`